.orchestra/config/components/python.yml

```diff
@@ -0,0 +1,60 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/make.lib.yml", "make")
+
+#@yaml/text-templated-strings
+---
+#@ def python_component():
+#@ source_url = "https://www.python.org/ftp/python/3.10.6/Python-3.10.6.tar.xz"
+license: LICENSE
+builds:
+  default:
+    configure: |
+      mkdir -p "$BUILD_DIR"
+      extract.sh --into "$BUILD_DIR" (@= source_url @)
+      cd "$BUILD_DIR"
+      
+      # orchestra's libc has a buggy getaddrinfo; we expect users to have a newer libc
+      ac_cv_buggy_getaddrinfo=no ./configure \
+        --prefix="$ORCHESTRA_ROOT" \
+        --libdir="${ORCHESTRA_ROOT}/lib64" \
+        --enable-optimizations \
+        --enable-shared \
+        --enable-ipv6 \
+        --disable-test-modules \
+        --with-ensurepip=install \
+        --with-lto \
+        --with-readline \
+        --with-system-ffi \
+        --with-system-expat \
```

Comment:
Why did you choose to provide expat instead of using the builtin one?

Reply:
Because we're going to use it for gdb anyway.